roadmap/implementers-guide/src/protocol-approval.md

```diff
@@ -38,7 +38,7 @@ We need two separate keys for the approval subsystem:
 
 - **Approval assignment keys** are sr25519/schnorrkel keys used only for the assignment criteria VRFs.  We implicitly sign assignment notices with approval assignment keys by including their relay chain context and additional data in the VRF's extra message, but exclude these from its VRF input.  
 
-- **Approval vote keys** would only sign off on candidate parablock validity and has no natural key type restrictions.  We could reuse the ed25519 grandpa keys for this purpose since these signatures control access to grandpa, although distant future node configurations might favor separate roles.
+- **Approval vote keys** would only sign off on candidate parablock validity and has no natural key type restrictions. There's no need for this to actualy embody a new session key type. We just want to make a distinction between assignments and approvals, although distant future node configurations might favor separate roles. We re-use the same keys as are used for parachain backing in practice.
```

Comment:
Assignments and approvals need separate keys so that assignments keys can stay on the machine while approvals keys might live on a remote signer.  

We employ the same keys for all candidate validity and invalidity statements (approvals, backing, and disputes)  because logically these incur similar risks.

Reply:
Is there a change in the text you want me to make?

Reply:
It's fine I guess.  It just confused me on a first reading.  What should we actually call them?  (candidate) validation keys?

Reply:
Yeah, I called them `para_validation` keys in the runtime description.